ch12.md

```diff
@@ -0,0 +1,256 @@
+# Chapter 11: Traversing the Stone
+
+So far, in our cirque du conteneur, you've seen us tame the ferocious [functor](ch8.md#my-first-functor), bending it to our will to perform any operation that strikes our fancy. You've been dazzled by the juggling of many dangerous effects at once using function [application](ch10.md) to collect the results. Sat there in amazement as containers vanished in thin air by [joining](ch9.md) them together. At the side effect sideshow, we've seen them [composed](ch8.md#a-spot-of-theory) into one. And most recently, we've ventured beyond what's natural and [transformed](ch11.md) one type into another before your very eyes.
+
+And now for our next trick, we'll look at traversals. We'll watch types soar over one another as if they were trapeze artists holding our value in tact. We'll reorder effects like the trolleys in a tilt-a-whirl. When our containers get intertwined like the limbs of a contortionist, we can use this interface to straighten things out. We'll witness different effects with different orderings. Fetch me my pantaloons and slide whistle, let's get started.
```

Comment:
> as if they were trapeze artists holding our value ***in tact***. 

Should *in tact* be *intact*. Or are you making reference to the heated debate all over the internets about the etymology.

Reply:
💯